README.rst

```diff
@@ -81,17 +88,28 @@ Assuming you named your database ``qwat``, edit the ``.pg_service.conf`` file an
 
 ::
 
-    # Qwat service name
     [qwat]
     #enter your database ip
-    host=192.168.0.1
+    host=127.0.0.1
+    #database name
+    dbname=qwat
+    port=5432
+    user=postgres
+    #you can also add your password if you like
+    password=YourPassword
+
+    [qwat_prod]
+    #enter your database ip
+    host=127.0.0.1
     #database name
     dbname=qwat
     port=5432
     user=postgres
     #you can also add your password if you like
     password=YourPassword
 
+Note: since version 1.3, another service definition is necessary to let the postgreSQL Update Manager initialize and migrate database versions.
```

Comment:
Are you sure that two services are required? This does not make too much sense to me.

Reply:
I haven't yet check this assertion. Was added by @haubourg 15 days ago 1376a47f6fdc27ff9a2b3eff3d50a78c2f138dd4

Reply:
Yes, needs more services.
Maybe we should get it out of here because we're talking about installation. This is linked to an update and there is a dedicated part for that. What do you think about that?

Reply:
Yes, I would not overwhelm the user with things that are not required at this point. And I find this confusing to the user.